content/cpp/concepts/math-functions/terms/sqrt/sqrt.md

```diff
@@ -0,0 +1,55 @@
+---
+Title: 'sqrt()'
+Description: 'Returns square root of the argument'
+Subjects:
+  - 'Computer Science'
+Tags:
+  - 'Functions'
+  - 'Arithmetic'
+CatalogContent:
+  - 'learn-c-plus-plus'
+  - 'paths/computer-science'
+---
+
+Returns square root of the argument
```

Comment:
```suggestion
The **`sqrt()`** function returns the square root of the argument.
```

Contrary to the current state/formatting of existing terms under [C++: Math Functions](https://www.codecademy.com/resources/docs/cpp/math-functions), the suggestions here aim for a new formatting, including:

- Beginning the entry by referring to the term by name.
- **Bolding** the first mention of the concept or term.